doc/tutorials/board_inspector_tool.rst

```diff
@@ -58,19 +59,25 @@ Board Inspector with the ``-h`` option yields the following usage message:
 .. code-block::
 
    usage: board_inspector.py [-h] [--out OUT] [--basic] [--loglevel LOGLEVEL]
-                [--check-device-status] board_name
+                             [--check-device-status]
+                             [--add-llc-cat <capacity_mask_length:int>,<clos_number:int>,<has_CDP:bool>]
```

Comment:
How about we also spell out CDP in the doc)?

Reply:
Amy just did a cut-and-paste here from the help output.  We could add a note about what CDP is I suppose, something like this after line 121 below

CDP (Code and Data Prioritization) performance feature enables separate control over code and data placement in the processor's L2 cache.

Reply:
Done